src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -1103,35 +1103,27 @@
      * </pre>
      * <p>parses as:</p>
      * <pre>
-     * +--CASE_GROUP
-     *     |
-     *     +--LITERAL_CASE (case)
-     *         |
-     *         +--EXPR
-     *             |
-     *             +--NUM_INT (0)
-     *     +--LITERAL_CASE (case)
-     *         |
-     *         +--EXPR
-     *             |
-     *             +--NUM_INT (1)
-     *     +--LITERAL_CASE (case)
-     *         |
-     *         +--EXPR
-     *             |
-     *             +--NUM_INT (2)
-     *     +--SLIST
-     *         |
-     *         +--EXPR
-     *             |
-     *             +--ASSIGN (=)
-     *                 |
-     *                 +--IDENT (x)
-     *                 +--NUM_INT (3)
-     *         +--SEMI (;)
-     *         +--LITERAL_BREAK (break)
-     *             |
-     *             +--SEMI (;)
+     * |--CASE_GROUP -&gt; CASE_GROUP
```

Comment:
first column is redundant, please remove it and shift left like in `TokenTypes#VARIABLE_DEF`